test/end-to-end.js

```diff
@@ -970,6 +972,36 @@ describe('end-to-end', function() {
       });
     });
 
+    it('passes scaffolded tests in development mode', function(done) {
+      var app = loopback({ localRegistry: true, loadBuiltinModels: true });
+      var bootOptions = {
+        appRootDir: SANDBOX,
+        env: 'development'
+      };
+      boot(app, bootOptions, function(err) {
+        app.listen(0, function() {
+          execNpm(['test'], { cwd: SANDBOX }, function(err, stdout, stderr) {
+            done(err);
```

Comment:
Here you need to make that request I mentioned in my earlier comment. See also what I wrote in https://github.com/strongloop/loopback-workspace/pull/284#discussion_r66055596.